frontend/packages/ceph-storage-plugin/src/__tests__/flexible-scaling.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { isFlexibleScaling } from '../utils/install';
+
+describe('isFlexibleScaling', () => {
+  it('return truthy when atleast 3 nodes and no AZ is passed', () => {
+    expect(isFlexibleScaling(4, 0)).toBe(true);
+  });
+  it('return truthy when atleast 3 nodes and 1 AZ is passed', () => {
+    expect(isFlexibleScaling(4, 1)).toBe(true);
+  });
+  it('return truthy when atleast 3 nodes and 2 AZ is passed', () => {
+    expect(isFlexibleScaling(4, 2)).toBe(true);
```

Comment:
I'll rather test 3 & 4 nodes both.
```suggestion
    expect(isFlexibleScaling(3, 2)).toBe(true);
```

Reply:
sounds good